ui/src/app/components/help-sidebar.tsx

```diff
@@ -237,6 +248,25 @@ const icons = (helpContentKey: string) => {
       style: {fontSize: '20px', marginLeft: '3px'},
       tooltip: 'Annotations',
     }];
+  if (
+    enableCustomRuntimes
+    && (
+      workspaceAccessLevel === WorkspaceAccessLevel.OWNER
```

Comment:
nit: can use this `canWrite` util: https://github.com/all-of-us/workbench/blob/28f2caa6c2d62038b2d7f1db9a68cecfc9862588/ui/src/app/utils/workspace-permissions.ts#L4